src/android/aidl/types/parcelables.md

```diff
@@ -0,0 +1,38 @@
+# Parcelables
+
+Binder for Rust supports sending parcelables directly:
+
+**birthday_service/aidl/com/example/birthdayservice/BirthdayInfo.aidl**:
+
+```java
+{{#include ../birthday_service/aidl/com/example/birthdayservice/BirthdayInfo.aidl}}
+```
+
+**birthday_service/aidl/com/example/birthdayservice/IBirthdayService.aidl**:
+
+```java
+import com.example.birthdayservice.BirthdayInfo;
+
+interface IBirthdayService {
+{{#include ../birthday_service/aidl/com/example/birthdayservice/IBirthdayService.aidl:with_info}}
+}
+```
+
+**birthday_service/src/client.rs**:
+
+```rust,ignore
+fn main() {
+    binder::ProcessState::start_thread_pool();
+    let service = connect().expect("Failed to connect to BirthdayService");
+
+{{#include ../birthday_service/src/client.rs:wish_with_info}}
+}
+```
+
+<details>
+
+- Parcelables work similarly to serde types in the broader Rust ecosystem: You
```

Comment:
IMHO they're mostly like C (or Rust) structs.

Reply:
Good point. I think the comment here is more confusing than helpful so I've removed it.